helm/charts/nats/values.yaml

```
@@ -6,6 +6,12 @@ global:
     # global image pull policy to use for all container images in the chart
     # can be overridden by individual image pullPolicy
     pullPolicy:
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistrKeySecretName
```

Prefer setting to empty value as opposed to comment for syntax highlighting, and so you can search for the value on artifact hub

```suggestion
    # global image pull secrets to use for all pod templates in the chart
    # https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
    pullSecrets: []
```